packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPickerUtils/SvgParserHelper.tsx

```diff
@@ -30,3 +46,20 @@ export const colorPickerPreviewSvg = (color: string): JSX.Element => {
     </svg>
   );
 };
+
+function adjustIconSize(element: Element, iconWidth: number | undefined, iconHeight: number | undefined) {
```

Comment:
comments for what this helper is doing with some examples?

Reply:
next pr

Reply:
Done